charts/ingress/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+The 3.x `kong` chart consolidates the functionality provided by the `ingress`
+and 2.x `kong` charts into a single chart. `ingress` users should upgrade to
+`kong` 3.x following the [upgrade guide](https://github.com/Kong/charts/blob/main/charts/kong/UPGRADE.md#300).
```

Comment:
As we don't know for sure what the shape will be in 3.0 - I'd rather not imply that the chart name will change for Ingress.

```suggestion
The 3.x `kong` chart consolidates the functionality provided by the `ingress`
and 2.x `kong` charts into a single chart. `ingress` users should upgrade to
3.x following the [upgrade guide](https://github.com/Kong/charts/blob/main/charts/kong/UPGRADE.md#300).
```